### ISSUE TITLE
Dashboard broken ?

### ISSUE THREAD

**Issue by softlion** [beefyfinance/beefy-dashboard#18]
![image](https://user-images.githubusercontent.com/190756/131340054-d22b1f57-a17f-4db1-8f58-611f02ccfeaa.png)

**Comment by kexleyBeefy** [comment]
You have to be signed in to a supported wallet to access the dashboard. The page uses whichever chain your wallet is on currently to display the relevant vaults. No chain = no vaults.

**Comment by softlion** [comment]
yes, the page stays blank if no wallet extension is active. I have activated the extension for this site and it works correctly now !
